src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+* {
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document

Reply:
don't apply anything except box-sizing to *